client/src/app/modules/events/event-card/event-card.component.html

```diff
@@ -1,16 +1,24 @@
-<section class="row mt-2 mb-2 w-100 px-0 mx-0" >
+<section class="row mt-2 mb-2 w-100 px-0 mx-0">
   <div class="col-sm-12 px-0">
-    <div class="card bg-primary text-white" (mouseenter)="toggleButtonMenu()" (mouseleave)="toggleButtonMenu()">
```

Comment:
I'm not big fan of telling here, in markup, what kind of color it should be... I moved it to CSS

Reply:
I personally dont mind, but bg-primary holds the theme color. we will have to make sure, that when we update theme, we also update everywhere else the $blue color